app/views/organizations/index.html.erb

```diff
@@ -37,6 +38,15 @@
             <td><%= org.neighborhood %></td>
             <td><%= link_to(org.web, org.web) if org.web.present? %></td>
             <td><%= org.members.count %></td>
+            <% if current_user %>
```

Comment:
as per this comment https://github.com/coopdevs/timeoverflow/pull/661/files#r997411162, I'd move the condition inside the `<td>`